Tests/SwiftFormatPrettyPrintTests/ExtensionDeclTests.swift

```diff
@@ -316,8 +316,8 @@ final class ExtensionDeclTests: PrettyPrintTestCase {
     let expected =
 
       """
-      public extension MyContainer: MyContainerProtocolOne,
-        MyContainerProtocolTwo,
+      public extension MyContainer:
```

Comment:
Not in this PR, but we should have an internal sanity check that none of the expected output lines are longer than the given line length (with an override for comments/string literals, I guess), since that would have caught this mistake—53 columns but line length of 50.